tools/hack/create-cluster.sh

```diff
@@ -2,17 +2,24 @@
 
 set -euo pipefail
 
-## Create kind cluster.
-tools/bin/kind create cluster \
-    --name envoy-gateway 
+# Setup default values
+CLUSTER_NAME=${CLUSTER_NAME:-"envoy-gateway"}
```

Comment:
we seem to have used `VAR ?= val` style in the Makefiles here e.g. https://github.com/envoyproxy/gateway/blob/d99f32feef8ddd6ede45bd0c93b607586138d4c8/tools/make/kube.mk#L2
prefer if we  use that style here as well

Reply:
1. this lines have same style as `run-kube-local.sh`
2. is `?=` valid in shell?

Reply:
oh yah my bad .....